docker-container/README.md

```diff
@@ -0,0 +1,61 @@
+# Based On:
+
+[https://github.com/alexcheng1982/docker-magento](https://github.com/alexcheng1982/docker-magento)
+
+With M1 being an older service there is no official support from the platform. Through my research this was the best overall solution giving a full breakdown of the process.
+
+# Dependencies
+
+## ngrok
+
+[https://ngrok.com/](https://ngrok.com/pricing)
+[https://ngrok.com/pricing](https://ngrok.com/pricing)
+
+In order for your docker container to communicate with the webhook and shipping & tax url set on [merchant.bolt.com](http://merchant.bolt.com/), one must set up an ngrok account to expose your store's docker ports to a public IP address. In order to do this one must have at least the Basic plan with ngrok to reserve a domain for themselves. 
+
+Once the initial configuration scripts are ran in your local environment and a public domain is reserved, run the following command 
+
+    ./ngrok http <STORE_PORT> -subdomain=<NGROK_DOMAIN>
```

Comment:
ngrok has a cool yml feature, you could add it in this folder and ask people to copy it to their local and use it.

Also on a side note we should pay for a team ngrok account since i think itll be useful for more than just this. lets chat tomorrow, 

starting the ngrok tunnel (once we have a paid account, and link it to a domain we own say `store.ethan.bolt-dev.com`) will become a simple command and maybe we can get rid of the change-hostname bit! lets chat tomorrow

Reply:
maybe we can use bolt.me for this? we use bolt.me for all internal sites eg., http://magento1-sandbox.bolt.me

Reply:
I was able to get this working with ethan-m1.dev.bolt.me. I need to work with Mike to standardize and document this process for the rest of the team.

Reply:
one thing you might want to think about:

if we use `*.ethan.dev.bolt.me` you will need to add only 1 record on ngrok for each developer. `m1.ethan...`, `shopify.ethan...`, ...